<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1931</width>
    <height>825</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>691</width>
     <height>621</height>
    </rect>
   </property>
   <property name="title">
    <string>Video1</string>
   </property>
   <widget class="QLabel" name="image_label_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>651</width>
      <height>571</height>
     </rect>
    </property>
    <property name="text">
     <string>DISPLAY1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>10</y>
     <width>691</width>
     <height>621</height>
    </rect>
   </property>
   <property name="title">
    <string>Video2</string>
   </property>
   <widget class="QLabel" name="image_label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>651</width>
      <height>571</height>
     </rect>
    </property>
    <property name="text">
     <string>DISPLAY2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>1410</x>
     <y>20</y>
     <width>501</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>Detect Pose</string>
   </property>
   <widget class="QPushButton" name="load_video_btn1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Video1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="load_video_btn2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Video 2</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>291</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>291</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="compare_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Compare</string>
    </property>
   </widget>
   <widget class="QPushButton" name="detect_pose_btn1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Detect Pose 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="detect_pose_btn2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Detect Pose 1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox1">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>RGB+Skeleton</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>130</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>RGB+Skeleton</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox1_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>80</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Skeleton</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox2_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>170</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Skeleton</string>
    </property>
   </widget>
  </widget>
  <widget class="QSlider" name="frame_no_slider">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>660</y>
     <width>1371</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="frame_no_slider_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>710</y>
     <width>1371</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>1410</x>
     <y>300</y>
     <width>511</width>
     <height>501</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Distance</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>491</width>
       <height>451</height>
      </rect>
     </property>
     <property name="title">
      <string>Plot</string>
     </property>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>481</width>
        <height>411</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout"/>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Visualize Embedding</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>481</width>
       <height>451</height>
      </rect>
     </property>
     <property name="title">
      <string>Plot</string>
     </property>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>471</width>
        <height>421</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2"/>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
